src/components/genre-section/genre-section.test.js

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import { GenreSection } from './genre-section';
+
+jest.mock('../../hooks/use-get-params/use-get-params', () => ({
+  useGetParams: jest.fn(() => ([{ filter: 'all' }])),
+}))
+
+const mockGenres = [
+  { id: 0, name: '0' },
+  { id: 1, name: '1' },
+  { id: 2, name: '2' },
+  { id: 3, name: '3' },
+]
+
+describe('Genre section', () => {
+  it('Checks "All" tab by default', () => {
```

Comment:
You don't have 'All' in the mocks, so it's a weird name for a test. 
There are a couple of options:
- It can check that the first Tab is set by default.
- You can change values in mocks, which will be similar to real Tabs data.

Reply:
Changed the description to **Checks the first tab by default**.